src/util/mod.rs

```diff
@@ -149,6 +150,142 @@ impl VersionTriple {
     }
 }
 
+#[derive(Debug, Error)]
+pub enum VersionDoubleError {
+    #[error("Failed to parse major version from {version:?}: {source}")]
+    MajorInvalid {
+        version: String,
+        source: std::num::ParseIntError,
+    },
+    #[error("Failed to parse minor version from {version:?}: {source}")]
+    MinorInvalid {
+        version: String,
+        source: std::num::ParseIntError,
+    },
+    #[error(
+        "Failed to parse version string {version:?}: string must be in format <major>[.minor]"
+    )]
+    VersionStringInvalid { version: String },
+}
+
+// Generic version triple
+#[derive(Clone, Copy, Debug, Eq, Ord, PartialEq, PartialOrd)]
+pub struct VersionDouble {
+    pub major: u32,
+    pub minor: u32,
+}
+
+impl Default for VersionDouble {
+    fn default() -> Self {
+        Self { major: 9, minor: 0 }
```

Comment:
`VersionDouble` is general purpose and not iOS-specific; see the precedent of `DEFAULT_MIN_SDK_VERSION` for the pattern of putting a const at the top of the `config/mod.rs` for the respective platform.